README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.com/vortexntnu/manta-auv.svg?branch=master)](https://travis-ci.com/vortexntnu/manta-auv)
```

Comment:
Is this icon only showing status for master branch regardless of which branch we are looking at?
Is it still running the check before merging any branch into master? Any way to test that before this is merged?

Reply:
There's no support for conditionals/variables in markdown, so unfortunately there's no way to change the link based on the current branch.

But pull requests are run through CI before merging, so that would be the easiest way to check if a branch passes the tests.